### ISSUE TITLE
Feature request: ability to prefix ECS repositories

### ISSUE THREAD

**Issue by Napas** [pgarbe/cdk-ecr-sync#414]
Hi,

I would like to request a feature which would allow prefixing all ECS repositories. This would allow separating our own repositories from the ones mirrored from the Docker Hub

I.e.
If prefix is set to `dockerhub-mirror`
* repo `nginx` in ECS would be `dockerhub-mirror/nginx`
* repo `localstack/localstack` in ECS would be `dockerhub-mirror/localstack/localstack`
* etc.

**Comment by Napas** [comment]
I've tried it today and it does create correct repositories, but still trying to upload images to the repositories without the prefix:

```
redacted is not authorized to perform: ecr:InitiateLayerUpload on resource: arn:aws:ecr:eu-west-1:redacted:repository/nginx
```

instead of using  `arn:aws:ecr:eu-west-1:redacted:repository/dockerhub-mirror/nginx`

**Comment by pgarbe** [comment]
You're right. I'm working on a fix

**Comment by pgarbe** [comment]
Should be fixed in v0.5.5

**Comment by Napas** [comment]
It works now, thank you!
